src/managers/SubscriptionManager.ts

```diff
@@ -527,34 +500,30 @@ export class SubscriptionManager {
             Because of this, we should unsubscribe the user from push first and then resubscribe
             them.
           */
-          await existingPushSubscription.unsubscribe();
 
           /* We're unsubscribing, so we want to store the created at timestamp */
-          shouldRecordSubscriptionCreatedAt = false;
+          isNewSubscription = await SubscriptionManager.doPushUnsubscribe(existingPushSubscription);
```

Comment:
talked in person, don't care about the result anymore. will have another check inside of `doPushSubscribe`